messages/en-US/global.json

```diff
@@ -105,5 +105,21 @@
   },
   "Back": {
     "btnBack": "Back"
+  },
```

Comment:
I know that we can do some clean up of these translation files eventually, but for now, I just created a separate 'Global' section in `global.json`.